mage/k8s/lib.go

```diff
@@ -0,0 +1,260 @@
+// Copyright DataStax, Inc.
```

Comment:
A large portion of the code in this file has been directly ripped out of `./mage/kind/lib.go`